qa/tasks/vstart_runner.py

```diff
@@ -261,22 +261,27 @@ def close(self):
         return FakeStdIn(self)
 
 
-class LocalRemote(object):
+class LocalRemote(RemoteShell):
     """
     Amusingly named class to present the teuthology RemoteProcess interface when we are really
     running things locally for vstart
 
     Run this inside your src/ dir!
     """
 
-    os = Remote.os
-    arch = Remote.arch
-
     def __init__(self):
+        super().__init__()
         self.name = "local"
-        self.hostname = "localhost"
+        self._hostname = "localhost"
         self.user = getpass.getuser()
 
+    @property
+    def hostname(self):
+        if not hasattr(self, '_hostname'):
```

Comment:
`_hostname` is always set in the constructor, do we actually delete this attribute afterwards ?

Reply:
Maybe. I haven't seen it AFAIR. The idea while writing this was to keep it as close as possible to teuthology.orchestra.remote.Remote.hostname.